src/Utils/Constants/dict.ts

```diff
@@ -0,0 +1,5 @@
+export interface I_DICT {
+    [key: string]: {
+        [key: string]: string; 
+    };
+};
```

Comment:
Maybe Fred or Ralph will correct me, but I think this should be typed how you are using it in the component. The component doesn't utilize variable I_DICT props so I don't see why you wouldn't strongly type the props to match how you are using them.

Reply:
I believe the typing for this can be a bit more strict with the keys Sun - Saturday being explicitly defined but because you have an 0-infinite length of categories it will be impossible to type the root level.